autogpt_platform/backend/backend/blocks/wavespeed/ai_media_generator.py

```
@@ -0,0 +1,202 @@
+            # Store the generated media to the user's workspace for persistence
+            stored_urls = [
+                await store_media_file(
+                    file=MediaFileType(url),
+                    execution_context=execution_context,
+                    return_format="for_block_output",
+                )
+                for url in output_urls
+            ]
+            yield "media_url", stored_urls[0]
+            yield "media_urls", stored_urls
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)